docs/source/nlp/usage_guides/nlp_properties_extended.rst

```diff
@@ -0,0 +1,77 @@
+.. _nlp__properties_ext:
+
+==========
+Properties
+==========
+
+Properties are one-dimension values that are extracted from the text. For example, a property could be as simple as
+**text length** or **sentiment**, or more complex properties such as the probability that the LLM avoided answering
+a question.
```

Comment:
I'm confused. You're describing there properties that are not in that repository.
Also, why choose these specific ones?

Reply:
Not sure what's the purpose of this doc, and it doesn't explain it as well.